src/cloudant/database.py

```diff
@@ -152,15 +152,12 @@ def create_document(self, data, throw_on_exists=False):
             corresponding to the new document in the database.
         """
         docid = data.get('_id', None)
-        doc = None
         if docid and docid.startswith('_design/'):
             doc = DesignDocument(self, docid)
         else:
             doc = Document(self, docid)
-        if throw_on_exists and doc.exists():
```

Comment:
I've removed the `.exists()` call (which adds an additional `HEAD` request) in favour of just catching the `409` on create. Does this sound reasonable?